package.json

```diff
@@ -71,25 +71,25 @@
     "babel-cli": "6.26.0",
     "babel-core": "6.26.0",
     "babel-preset-env": "1.6.1",
-    "body-parser": "1.17.1",
+    "body-parser": "1.18.2",
     "conventional-changelog-lint": "1.1.9",
-    "coveralls": "2.13.0",
+    "coveralls": "3.0.0",
```

Comment:
I'll check all upgrades from the licensing standpoint, but under this one I can comment right away.

`coveralls` do not have a license distributed with the package, nor they have the license in the GitHub repo. I tried to contribute a solution - https://github.com/nickmerwin/node-coveralls/pull/175, but there's no activity. When trying to figure out the copyright notice, I was surprised to find out the company uses two different names: "Coveralls, LLC" (on their site) and "LEMUR Heavy Industries" ([lemurheavy](https://github.com/lemurheavy) - author organization of the GitHub integration apps). There is no contact page on the coveralls.io website, no way to figure out who actually runs the business. Maybe it's just a lack of organization, maybe sloppiness, but it doesn't really build trust.

At the same time, the library itself is at @nickmerwin's personal GitHub profile, and the `author` field in `package.json` mentions someone called "Gregg Caines". The `maintainers` field mentions `nick@coveralls.io`, so I suppose it's not Nick's "hobby" project. Anyway, it's a mess. And I've seen a lot of npm packages, but this is one of the messiest I've seen.

I've tried to contact the maintainers over the GitHub issues and by email. No response. At about the same time (not implying anything though) the integration stopped to work under Dredd's Pull Requests. I tried to debug the issue like 10 times with no success. I tried to contact coveralls.io, no response.

Honestly, I think I'm giving up on Coveralls. While I believe the test coverage is very important, I also believe there are Coveralls competitors and we should go with them. So if I may, I'd kindly ask you not to upgrade the `coveralls` dependency and if you have a spare time, a Pull Request removing coveralls from the toolchain would be awesome.

I discovered following alternatives:

- https://www.codacy.com/
- https://codeclimate.com/
- https://codecov.io/

Do you have any experience with any of them?

Reply:
Ouch, that’s some deep looking you did I to that. I’ll revert that part of the PR.

I can recommend codecov.io, it’s the solution we use at Contentful and IMHO it’s much better hab coveralls.

Reply:
Yeah, I really tried hard to fix it and to continue using Coveralls before I gave up 😄

Reply:
Removed the coveralls update. I'd prefer not to do the removal PR, IMHO that should happen in one go with the set up of a new solution which I can't do.